psqlgraph/psql.py

```diff
@@ -235,25 +263,22 @@ def _configure_driver_mappers(self):
                 'Have you imported your models?'
             ).format(str(e)))
 
-    def voided_nodes(self, query=VoidedNode):
+    def voided_nodes(self, query=None):
+        query = query or VoidedNode
```

Comment:
What's the difference before and after the change?

Reply:
Removes code duplication